src/components/Details/TranscriptSearch/TranscriptItems.tsx

```diff
@@ -70,11 +65,7 @@ const TranscriptItems: FC<TranscriptItemsProps> = ({
             searchQuery={searchQuery}
             speakerId={sentences[index].speaker_id}
             speakerPictureSrc={sentences[index].speaker_pictureSrc}
-            handleJumpToTranscript={handleJumpToTranscript(
-              index,
-              sentences[index].session_index,
-              sentences[index].start_time
-            )}
+            handleJumpToTranscript={handleJumpToTranscript(sentences[index].index)}
```

Comment:
I think this is the bugfix right here right?

Reply:
`handleJumpToTranscript={handleJumpToTranscript(sentences[index].index)}`
Yep, use the sentence pre-calculated index instead of the index relative to the list of visible sentences.